sdk/loadtestservice/azure-developer-loadtesting/src/test/resources/session-records/LoadTestRunTests.listMetrics.json

```diff
@@ -38,7 +38,7 @@
       "mise-correlation-id" : "106f0fe1-bdee-4f99-9b27-664a5ea53c40",
       "retry-after" : "0",
       "StatusCode" : "200",
-      "Body" : "{\"timeseries\":[{\"data\":[{\"timestamp\":\"2022-12-09T07:11:00.000Z\",\"value\":1.0}]}]}",
+      "Body" : "{\"value\":[{\"data\":[{\"timestamp\":\"2022-12-09T07:11:00.000Z\",\"value\":1.0}]}]}",
```

Comment:
It is change to backend?

I assume backend cannot change like this after GA?

Reply:
Yeah, this is change in backend. REST API spec PR is also updated to match the same. Our service is not yet GA, hence we can change for now